packages/docs/community/release-process/base-cli-contractkit-dappkit-utils.md

```diff
@@ -0,0 +1,81 @@
+All releases should be tagged with the version number, e.g. `contractkit-vX.Y.Z`. Each release should include a summary of the release contents, including links to pull requests and issues with detailed description of any notable changes. 
+
+### Communication
+
+The community will be notified of package updates through the following channels: 
+
+For all releases: 
+  * Each package’s `CHANGELOG.md` file, as mentioned above
+  * Github releases page, as mentioned above
+  * [Discord](https://discord.gg/6yWMkgM): #developer-chat, #mainnet, and #sdk
+
+For major releases: 
+  * Twitter: [@CeloDevs](https://twitter.com/CeloDevs)
+  * Mailing list: cLabs’ Tech Sync
+  * [Celo Forum](https://forum.celo.org/)
+
+
+
+## Build process
+
+The packages are published by following the instructions [here](https://github.com/celo-org/celo-monorepo/blob/master/README-dev.md).
+
+## Testing
+All builds of these packages are automatically tested for performance and backwards compatibility in CI. Any regressions in these tests should be considered a blocker for a release.
+Minor and major releases are expected to go through additional rounds of manual testing as needed to verify behavior under stress conditions.
+
+{% hint style="warning" %}
+Work in progress
+{% endhint %}
+
+## Promotion process 
+  * For a patch release: The first commit of this process should change the version number encoded in the source from `x.y.z` to `x.y.z+1-dev` and the final commit should change the published version number from `x.y.z-1`  to `x.y.z`.
+  * For minor releases, the same process should be followed, except the `y` value would increment, and the `z` value would become 0. 
+  * For major releases, the same process should be followed, except the `x` value would increment, and `y` and `z` values would become 0. 
+
+Only one commit should ever have a non-dev tag at any given version number. When that commit is created, a tag should be added along with release notes. Once the tag is published it should not be reused for any further release or changes.
+
+### Emergency patches
+Bugs which affect the security, stability, or core functionality of the network may need to be released outside the standard release cycle. In this case, an emergency patch release should be created on top of all supported minor releases which contains the minimal change and corresponding test for the fix.
```

Comment:
Do you think it would be useful to add something about publishing an emergency patch retro (why the patch was necessary and what it includes) for any releases that deviate from the published release process? If so, add it here?

Reply:
Ooh that's a good point! Do we have a format for that? If so, i can submit a new PR to add to this section, if that's okay

Reply:
Not that I am aware of. I think it would be fine to keep it super simple (ie what's in it and why)